packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -1,51 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`Notification component tests Notification default 1`] = `
```

Comment:
Why is this deleted?

Reply:
You can update the test with 'yarn test -u' at the end. But that makes sense.

Reply:
Ooh ok sounds good!